.github/workflows/build-release-weekly.yml

```diff
@@ -151,18 +168,18 @@ jobs:
     runs-on: windows-latest
     needs: create-release
     env:
-      QT_SELECT: 5
       UPLOAD_URL: "${{ needs.create-release.outputs.upload_url }}"
 
     steps:
       - uses: actions/checkout@v2
       - uses: ilammy/msvc-dev-cmd@v1
-      - name: install build dependencies
-        uses: jurplel/install-qt-action@v2
+      - uses: jurplel/install-qt-action@v2
```

Comment:
from ilammy and jurple1 repos?

Reply:
I've made a note about external actions in the README.md located within the .github/workflows directory.